bpdm-gate/src/main/resources/application.properties

```diff
@@ -28,17 +28,6 @@ bpdm.api.upsert-limit=100
 bpdm.bpn.agency-name=Catena-X
 bpdm.bpn.name=Business Partner Number
 bpdm.bpn.id=CX_BPN
-#Handle configuration of SaaS API connection
-bpdm.saas.host=${BPDM_SAAS_HOST:http://localhost:1234}
-bpdm.saas.storage=${BPDM_SAAS_STORAGE:storage_id}
-bpdm.saas.datasource=${BPDM_SAAS_DATASOURCE:datasource_id}
-bpdm.saas.legal-entity-type=LEGAL_ENTITY
-bpdm.saas.site-type=ORGANIZATIONAL_UNIT
-bpdm.saas.address-type=BP_ADDRESS
-bpdm.saas.api-key=${BPDM_SAAS_KEY:}
-bpdm.saas.data-exchange-api-url=/data-exchange/rest/v4/storages/${bpdm.saas.storage}
-bpdm.saas.data-clinic-api-url=/data-clinic/rest/storages/${bpdm.saas.storage}
-bpdm.saas.sharing-timeout=2d
```

Comment:
You are deleting the default values here for the properties. The actual config properties are still there as a data class in the code. Better keep those properties in for now and we make a separate issue/pull request to delete the necessary config properties from the code